packages/client/src/comp/states/Kansas.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { BaseInput } from '../util/Input'
+import { KansasInfo } from '../../common'
+import { SignatureBase, StatelessInfo, NoSignature } from './Base'
+import { Upload } from '../util/Upload'
+import { TogglableDropdown } from '../util/TogglableDropdown'
+
+const options = [
+  'Driver\'s License Number',
+  'Copy of Photo ID',
+] as const
+
+export const Kansas = () => {
+  const [idPhoto, setIdPhoto] = React.useState<string>()
+  const [idType, setIdType] = React.useState<'Number' | 'Image'>('Number')
+  const [idNumber, setIdNumber] = React.useState<string>()
+  const enrichValues = (baseInfo: StatelessInfo): NoSignature<KansasInfo> | null => {
+    return {
+      ...baseInfo,
+      state: 'Kansas',
+      idType,
+      idPhoto,
+      idNumber,
+    }
+  }
+
+  // Clears unused inputs
+  React.useEffect(() => {
+    if (idType === 'Image') {
+      setIdNumber(undefined)
+    } else {
+      setIdPhoto(undefined)
+    }
+  }, [idType, setIdNumber, setIdPhoto])
+
+  return <SignatureBase<KansasInfo>enrichValues={enrichValues}>
+    <TogglableDropdown<typeof options>
+      defaultValue={'Driver\'s License Number'}
+      label='Identification Method:'
+      style={{ marginTop: 40 }}
+      options={options}
+      onChange={value => setIdType(
```

Comment:
Instead of using `React.useEffect` to clear input, why not set it in `onChange`?